test/fixtures/api/attributes-named-type-enum-reference.json

```diff
@@ -227,7 +238,7 @@
                                   "content": "application/schema+json"
                                 }
                               },
-                              "content": "{\n  \"$schema\": \"http://json-schema.org/draft-04/schema#\",\n  \"type\": \"object\",\n  \"properties\": {\n    \"a\": {\n      \"type\": \"string\",\n      \"enum\": [\n        \"valueB\",\n        \"valueA\"\n      ]\n    }\n  },\n  \"required\": [\n    \"a\"\n  ]\n}"
+                              "content": "{\n  \"$schema\": \"http://json-schema.org/draft-04/schema#\",\n  \"type\": \"object\",\n  \"properties\": {\n    \"a\": {\n      \"type\": \"string\",\n      \"enum\": [\n        \"valueB\",\n        \"valueA\",\n        \"valueA\"\n      ]\n    }\n  },\n  \"required\": [\n    \"a\"\n  ]\n}"
```

Comment:
Is this a regression? the JSON Schema enum now contains duplicate `valueA`, perhaps this is going to be solved by @tjanc work on the JSON Schema visitor?

Reply:
Yes it probably need to be resolved in JSON schema visitor. I did not to want to touch it because Thomas is working on.

Reply:
This must be fixed earlier on, it's a regression.